phpunit.xml.dist

```diff
@@ -14,6 +14,7 @@
 	<testsuites>
 		<testsuite name="default">
 			<directory suffix="-test.php">./phpunit/</directory>
+			<directory suffix=".php">./phpunit/tests/</directory>
```

Comment:
WordPress Core does it this way https://github.com/WordPress/wordpress-develop/blob/trunk/phpunit.xml.dist#L17.

I left the original `<directory>` directive on purpose to:

* allow all other tests to run, until they are also moved into this Core specific `tests` directory.
* avoid disruption in existing PRs.